components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -115,6 +115,22 @@ sealed class TabListAction : BrowserAction() {
     object RemoveAllNormalTabsAction : TabListAction()
 }
 
+/**
+ * [BrowserAction] implementations related to updating the [TabSessionState] inside [BrowserState].
+ */
+sealed class TabSessionAction : BrowserAction() {
```

Comment:
nit: TabSessionAction is very generic and may be confusing if there's nothing else in here? Others like `TrackingProtectionAction` are more specific. I fear that this action group may end up becoming a dumping ground. :)

Reply:
This was my fear as well. I was hesitant to put this under `TabListAction` since it wasn't acting on the list, and it didn't fit into `ContentAction` as well since it isn't content.

Maybe I should change the name to something else, like `TabSelectionAction`? 🤔

Reply:
It's a bit of a special case, maybe just `LastAccessAction` and `UpdateLastAccessAction` to prevent from becoming a dumping ground?